### PR TITLE
aconcli: Added detailed help messages for each command

### PR DESCRIPTION
**Pull request by xxu36**

### CODE REVIEW

File: aconcli/cmd/export.go [hunk 1]
```diff
@@ -15,12 +15,13 @@ import (
...
+	Use:   "export <manifest file>",
+	Short: "Export the ACON image into a tarball file",
 	Long: `
-Export the specified ACON workloads into a tarball file, including the
-workload manifests, the certificates, the signatures and their file
-system layers`,
+Export the ACON image corresponding to the specified manifest file into
+a tarball. The image includes a manifest file, a signature file, a
+certificate file to verify the signature, and the file system layers
+of the image`,
```
Comment (binxing) [inline aconcli/cmd/export.go:24]: Please clarify if `export` (guess it doesn't) exports images on which the specified image/manifest depends.

A bit off topic here, what's the recommended way to figure out and export dependencies (i.e., images that a specified image depends on)?

File: aconcli/cmd/generate.go [hunk 2]
```diff
@@ -18,12 +18,13 @@ import (
 )
 
 var generateCmd = &cobra.Command{
-	Use:   "generate DOCKER_IMAGE",
-	Short: "Generate a manifest and commit file system layers to ACON repository",
+	Use:   "generate <Docker image>",
```
Comment (binxing) [inline aconcli/cmd/generate.go:21]: Here `<Docker image>` has the 1st letter capitalized but other (e.g., `<manifest file>`) place holders are all in lower case. It'd be better to have a consistent style.

Also, it looks like `git` style placeholders (lower case phrases surrounded by angle brackets) are being used here. If so, please use hyphens (`-`) in place of spaces (` `) for separating words.

File: aconcli/cmd/generate.go [hunk 3]
```diff
@@ -18,12 +18,13 @@ import (
...
+	Short: "Generate a manifest file and commit file system layers to ACON repository",
 	Long: `
-Generate a manifest file in JSON format and commit to ACON repository the file
-system layers from specified Docker image. The resulting manifest file can be
-further edited manually and then be signed using the 'sign' sub-command.`,
+Generate a manifest file in JSON format and commit to ACON repository
+the file system layers from specified Docker image. The output manifest
+file can be further edited manually and can be signed or resigned using
```
Comment (binxing) [inline aconcli/cmd/generate.go:26]: `re-signed` instead of `resigned`

File: aconcli/cmd/restart.go [hunk 4]
```diff
@@ -12,10 +12,13 @@ import (
 )
 
 var restartCmd = &cobra.Command{
-	Use:   "restart Container_ID",
+	Use:   "restart",
```
Comment (binxing) [inline aconcli/cmd/restart.go:15]: Is `<container-id>` still needed?

File: aconcli/cmd/generate.go (hunk 2 above)
Comment (xxu36) [reply aconcli/cmd/generate.go]: resolved

File: aconcli/cmd/restart.go (hunk 4 above)
Comment (xxu36) [reply aconcli/cmd/restart.go:15]: Not any more. container is now specified by the flag

File: aconcli/cmd/generate.go (hunk 3 above)
Comment (xxu36) [reply aconcli/cmd/generate.go]: resolved

File: aconcli/cmd/export.go (hunk 1 above)
Comment (xxu36) [reply aconcli/cmd/export.go]: Resolved. 

If in the manifest file, there are alias names which cannot be resolved locally (i.e. in current manifest file), then we have dependencies. Currently, the user needs to track which imported images are used as the dependencies.

**Pull request merged**